### ISSUE TITLE
suggestion: allow for organisation of search fiter fields

### ISSUE THREAD

**Issue by EZS-JD** [salesagility/SuiteCRM#2299]
#### Issue

enhancement suggestion:
Allow for some grouping of search filter fields to organize them better. We need many different search filter fields for various different departments and the way they're presented it's not easy to find the field you're looking for...
#### Steps to Reproduce

1.
2.
3.
4.
#### Your Environment

- SuiteCRM Version used: 7.4.4
- Browser name and version (e.g. Chrome Version 51.0.2704.63 (64-bit)):
- Environment name and version (e.g. MySQL, PHP 7):
- Operating System and version (e.g Ubuntu 16.04):

**Comment by murdal** [comment]
Could you explain your suggestion a bit further with what you mean by grouping?

Currently you could add HTML fields to act as headers for your search criteria and the security suite plug-in offers the functionality to have different layouts per team.

**Comment by samus-aran** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [suggestion: allow for organisation of search fiter fields](https://trello.com/c/QNByTmSU/72-suggestion-allow-for-organisation-of-search-fiter-fields)

**Comment by samus-aran** [comment]
We have now set up a new home for suggestions at Trello.  All github issues that were labeled 'suggestion' have been moved and will be closed.  Certain ones will be progressed within the new Suggestion Box and may be re-opened.

Announcement of moving Suggestions:
https://suitecrm.com/forum/suggestion-box/13691-moving-suggestions-from-github-to

New SuiteCRM Suggestion Box
https://trello.com/b/Ht7LbMqw/suitecrm-suggestion-box
